matveev_alexey/task3.cpp

```diff
@@ -103,9 +110,52 @@ class Graph {
   EdgeId getNewEdgeId() { return edge_new_id_++; }
 };
 
+class GraphPrinter {
+ public:
+  GraphPrinter(const Graph& graph) : graph_(graph) {}
+  std::string print() const {
+    std::string graph_table;
```

Comment:
Ни разу это не таблица, это либо строка, либо результат
```suggestion
    std::string graph_string;
```
```suggestion
    std::string result;
```